tests/tests_signin_user.py

```diff
@@ -0,0 +1,13 @@
+from django.test import Client
+
+
+def test_with_authenticated_client(client, django_user_model):
+	username = "test"
+	password = "test"
+	user = django_user_model.objects.create_user(username=username, password=password)
+	client.force_login(user)
+	respons = client.get('/personal_area', follow=True)
+	assert respons.status_code == 200
+	c = Client()
+	respons2 = c.post('/sign_in/', {'username': 'john', 'password': 'smith'})
+	assert respons2.content.find(b"run your job search now") == -1
```

Comment:
It means a negative test, Is it? 
since here it is passing if you do not find the personal search

Reply:
It's because I'm probably already connected

Reply:
the assert was wrong for the second user
You checked that assert respons2.content.find(b"run your job search now") == -1 not found by the 2nd query, which is not failing the test,
But is that what you really want to test?